### PR TITLE
feat: Add a pizza fact

### PR DESCRIPTION
**Pull request by prw760**
## Description

This PR adds a pizza fact

## What type of PR is this? (check all applicable)

- [ X] ☝️ Add a pizza fact or trivia
- [ ] 🧑‍🍳 Add a pizza recipe
- [ ] 🗺️ Add a regional pizza
- [ ] 🍴 Add a traditional food
- [ ] 📝 Documentation Update

## Related Tickets & Documents

Related to #36

## Added to documentation?

- [ ] 📜 README.md
- [ X] 🙅 no documentation needed

### CODE REVIEW

File: pizza-facts-&-trivia.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ We've categorized the facts into different sections to help explore the diverse
 - The world's largest pizza festival, known as "Pizza Fest," takes place annually in Naples, Italy, where pizza originated.
 - As of 2022, there were 80,175 pizza restaurants in the United States, an increase of almost 9,000 restaurants over the past decade.
 - Some pizzerias in Kenya offer pizzas with unique toppings, like nyama choma (grilled meat) and a spicy blend of Maasai herbs and spices.
+- "Pizza John’s Jardín Escondido" restaurant in El Castillo, Costa Rica, is widely considered the world’s most remote pizzaria. It is situated in the middle of the rainforest in Volcán Arenal national park -- and has fantastic views of mountains, lagoons, and wildlife. The pizzaria is located 80+ miles from the nearest city and approx 40 miles from the nearest small town.  Diners are encouraged to commemorate their visit by signing the pizzeria’s "wall of signatures."
```
Comment (adiati98) [inline pizza-facts-&-trivia.md:32]: **Nitpick**: Let's fix the capitalization, spacing, etc., here.

```suggestion
- "Pizza John’s Jardín Escondido" restaurant in El Castillo, Costa Rica, is widely considered the world’s most remote pizzaria. It is situated in the middle of the rainforest in Volcán Arenal National Park — and has fantastic views of mountains, lagoons, and wildlife. The pizzaria is located 80+ miles from the nearest city and approximately 40 miles from the nearest small town. Diners are encouraged to commemorate their visit by signing the pizzeria’s "wall of signatures."
```

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @prw760,

I left a nitpick review here. 🙂

You can simply click the "commit suggestion" button if you agree with the suggestion whenever you encounter sonething like this in the future.

Also, please mind the bot. I'm making a fix to our GitHub action to include "relate(d/s) to" keywords for linking issues. Your PR template is already good! 👍🏽 

Thanks!

**Comment by adiati98** [comment]
@prw760 we've merged a PR before yours. So, please [resolve the merge conflicts](https://github.com/open-sauced/pizza-verse?tab=readme-ov-file#resolving-merge-conflicts) before we review your PR.

Thank you! 🙂

**Comment by prw760** [comment]
Hi, Ayu.   I accepted your changes, but I cannot click the resolve conflicts area because I don't have write access to the main repo.  Do I have an actionable still -- or am I waiting for the repo managers?   Thanks again for your patience while I'm learning.

**Comment by adiati98** [comment]
Hey @prw760,

Answering your question, whenever you see merge conflicts in you PR, you need to resolve them yourself.
Most of the time, maintainers won't do this for you because they don't have the resources to do so.

I've actually gave you the link to resolve conflicts in my previous comment, but I'm dropping it [here](https://github.com/open-sauced/pizza-verse?tab=readme-ov-file#keeping-branch-up-to-date-and-resolving-merge-conflicts) again.

I wrote a [blog post](https://www.freecodecamp.org/news/keep-branches-up-to-date-resolve-merge-conflicts/) as well about resolving merge conflicts if you'd like to learn further. 🙂

**Comment by adiati98** [comment]
If you haven't deleted your local branch, I encourage you to try resolving the merge conflicts to practice. 😊

**Pull request closed without merging**